### PR TITLE
Ghosts will now see nearby messages in bold

### PR DESCRIPTION
**Pull request by ghost**
So you can separate out the messages the person you're orbiting is saying from other chatter

:cl: More Robust Than You
tweak: Ghosts will now see nearby messages in bold
/:cl:

### CODE REVIEW

**Comment by Repukan** [comment]
Awful, just toggle death ears if you want to see local chat

**Comment by KorPhaeron** [comment]
This should only happen if ghost ears are on

**Comment by Dorsisdwarf** [comment]
probbably one to be live-tested so we can see how it works out in a properly spammy round

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]
Make this only happen when ghost ear are on

**Comment by MoreRobustThanYou** [comment]
done

File: code/modules/mob/dead/observer/say.dm [hunk 1]
```diff
@@ -24,5 +24,7 @@
 	var/link = FOLLOW_LINK(src, to_follow)
 	// Recompose the message, because it's scrambled by default
 	message = compose_message(speaker, message_language, raw_message, radio_freq, spans, message_mode)
+	if((speaker in view(src)) && client && client.prefs && (client.prefs.chat_toggles & CHAT_GHOSTEARS))
```
Comment (Cyberboss) [inline code/modules/mob/dead/observer/say.dm]: prefs always exists

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
I don't like it

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]
You will have to refactor how this works if you want it to go in, we can't run view() for every message and ghost

**Comment by Cyberboss** [comment]
Also further modifying already potentially modified text smells terrible

**Comment by KorPhaeron** [comment]
Three maintainers said no in a row now so

**Pull request closed without merging**